masschallenge/impact-api
# [AC 5826] - Non-finalist People Profile pages should only be accessible to staff.

## [AC 5826](https://masschallenge.atlassian.net/browse/AC-5826)
**Changes Introduced**
- Finalist and mentors only have access to confirmed finalists profile pages
- Non-finalist People Profile pages are only accessible to staff.

**How to test**

1. Staff should be able to view all profiles: 
- [masquerade as a staff user](http://localhost:8181/admin/simpleuser/user/4668/masquerade/)
- View [finalist profile](http://localhost:1234/people/44670) {should be accessible}
- View [non finalist profile](http://localhost:1234/people/891) {should be accessible}
2. Finalists and mentors should only have access to confirmed finalists profiles
- [masquerade as a mentor/non staff user](http://localhost:8181/admin/simpleuser/user/845/masquerade/)
- View [finalist profile](http://localhost:1234/people/44670) {should be accessible}
- View [non finalist profile](http://localhost:1234/people/891) {should not be accessible}. Expected error message:  "Sorry, You are not allowed to access this page."
